src/action.js

```diff
@@ -25,6 +25,8 @@ const LANG_TO_EXTENSION = {
   'scala': 'scala',
   'swift': 'swift',
   'typescript': 'ts',
+  'racket': 'rkt',
+  'erlang':'erl',
```

Comment:
Thanks! Minor nit: do you mind keeping them alphabetically sorted and add a whitespace before 'erl' and after the colon? Feel free to add a comment on leaving it alphabetically sorted since there isn't one currently.

Looks good to me after that change!